src/main/java/org/gridsuite/study/server/service/ConsumerService.java

```diff
@@ -398,367 +239,199 @@ public Consumer<Message<String>> consumeCaseImportFailed() {
         };
     }
 
-    @Bean
-    public Consumer<Message<String>> consumeSensitivityAnalysisResult() {
-        return message -> {
-            UUID resultUuid = UUID.fromString(message.getHeaders().get(RESULT_UUID, String.class));
-            String receiver = message.getHeaders().get(HEADER_RECEIVER, String.class);
-            if (receiver != null) {
-                NodeReceiver receiverObj;
-                try {
-                    receiverObj = objectMapper.readValue(URLDecoder.decode(receiver, StandardCharsets.UTF_8), NodeReceiver.class);
-
-                    LOGGER.info("Sensitivity analysis result '{}' available for node '{}'", resultUuid, receiverObj.getNodeUuid());
-
-                    // update DB
-                    updateSensitivityAnalysisResultUuid(receiverObj.getNodeUuid(), resultUuid);
+    /**
+     * parse the error message from the computation microservice and uses its data to notify the front
```

Comment:
la fonction ne parse pas vraiment le message
```suggestion
     * Consumes a calculation failed message and notify the front based on the computation type.
```

Reply:
Changed.